datadog_checks_dev/datadog_checks/dev/tooling/signing.py

```diff
@@ -8,16 +8,16 @@
 
```

Comment:
`datadog_checks/dev/tooling/signing.py` is ignore in `pyproject.toml`, but looks like skip_glob is not working anymore with isort v5, see: https://github.com/PyCQA/isort/issues/1468

Reply:
So skip_glob seems to work but the pattern has to be different.
Why does this file needs to change if the skip_glob has been correctly updated?

Reply:
@FlorianVeaux Good catch. Forgot to revert this file after fixing skip_glob